app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -176,3 +172,8 @@ export default {
   },
 };
 </script>
+<style scoped>
+.export-radios ::v-deep .v-label {
+  margin-bottom: 0 !important;
+}
```

Comment:
If we're going to adjust the radio button vertical alignment issue it should be a global style setting. It's not just this export dialog with the issue
See "Vertical alignment of text for Checkboxes and Radio buttons (see Create New Form) too high." in https://apps.nrs.gov.bc.ca/int/jira/browse/SHOWCASE-1656

This is (another) issue introduced by Bootstrap it looks like. So if we do want to address it quickly with CSS overrides can apply a class to places it needs to be fixed globablly in style.scss maybe rather than scoped.

We should double check the radios and checkboxes on the actual FormIO forms themselves too.

Reply:
i've added a global style in my PR for vuetify checkbox and radio button labels, so this isn't needed.

Reply:
ok. thank you. i'll remove this and test.

Reply:
formio checkbox/radio labels look good on firefox and edge.